system/monitoring.rst

```diff
@@ -20,3 +23,9 @@ Zammad has issue (whatever nature, refer to image 2 for interface example)::
 
 .. image:: /images/system/monitoring/1.jpg
```

Comment:
I suggest to reorder this part as well. First should come the text and image where things are green, and then the part where there is an alert (or the other way round). Not first all text with "refer to image 2".